src/views/Home/components/CakeWinnings.tsx

```diff
@@ -2,24 +2,37 @@ import React from 'react'
 import { useTotalClaim } from 'hooks/useTickets'
 import { getBalanceNumber } from 'utils/formatBalance'
 import { usePriceCakeBusd } from 'state/hooks'
+import { Skeleton, Text } from '@pancakeswap-libs/uikit'
+import { useWeb3React } from '@web3-react/core'
 import { BigNumber } from 'bignumber.js'
 import styled from 'styled-components'
 import CardValue from './CardValue'
 import CardBusdValue from './CardBusdValue'
+import useI18n from '../../../hooks/useI18n'
```

Comment:
Please use absolute import

Reply:
done

Reply:
I will create an another PR for the rest of the imports that is not related for this pr